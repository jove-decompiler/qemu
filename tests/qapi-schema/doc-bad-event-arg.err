doc-bad-event-arg.json:3: the following documented members are not in the declaration: a
