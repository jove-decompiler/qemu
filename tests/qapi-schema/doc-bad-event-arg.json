# Arguments listed in the doc comment must exist in the actual schema
# BUG: nonexistent @a is not rejected

##
# @FOO:
# @a: a
##
{ 'event': 'FOO' }
