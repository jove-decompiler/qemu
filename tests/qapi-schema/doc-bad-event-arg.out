module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module doc-bad-event-arg.json
event FOO None
    boxed=False
doc symbol=FOO
    body=

    arg=a
a
